docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openjdk:8-jre-alpine
+FROM openjdk:12.0.2-jdk-oracle
```

Comment:
we use lts jdks only, switch to 11.

Also we do not use proprietary Oracle builds.